### PR TITLE
chore: recorder button styling

### PR DESCRIPTION
**Pull request by debs-obrien**
- improved styling based on GitHub paddings but with VS Code colors
- added a span for buttons with text and icon to easily add padding
- added type for button for better accessibility
- improved padding of wrapper for explore text

Screenshot 2022-12-02 at 13 26 40
Screenshot 2022-12-02 at 13 27 00

### CODE REVIEW

File: packages/web/src/components/toolbarButton.css [hunk 1]
```diff
@@ -16,15 +16,25 @@
...
+  white-space: nowrap;
+  vertical-align: middle;
+  cursor: pointer;
+  appearance: none;
+  user-select: none;
+  text-decoration: none;
+  text-align: center;
+  padding: 5px 8px;
+  font-size: 14px;
+  color: var(--vscode-sideBarTitle-foreground);
```
Comment (pavelfeldman) [inline packages/web/src/components/toolbarButton.css:37]: This is already defined.

File: packages/web/src/components/toolbarButton.css [hunk 2]
```diff
@@ -16,15 +16,25 @@
...
-  border: none;
-  outline: none;
   color: var(--vscode-sideBarTitle-foreground);
   background: transparent;
-  padding: 4px;
   margin-left: 10px;
-  cursor: pointer;
   display: inline-flex;
   align-items: center;
+  border-radius: 6px;
```
Comment (pavelfeldman) [inline packages/web/src/components/toolbarButton.css:24]: What are you aligning the visual language with?

File: packages/web/src/components/toolbarButton.css [hunk 3]
```diff
@@ -16,15 +16,25 @@
...
+  border: 1px solid var(--vscode-dropdown-border);
```
Comment (pavelfeldman) [inline packages/web/src/components/toolbarButton.css:25]: ditto

File: packages/web/src/components/toolbarButton.css [hunk 4]
```diff
@@ -16,15 +16,25 @@
...
+  font-family: inherit;
```
Comment (pavelfeldman) [inline packages/web/src/components/toolbarButton.css:26]: What are you resetting from?

File: packages/web/src/components/toolbarButton.css [hunk 5]
```diff
@@ -16,15 +16,25 @@
...
+  line-height: 20px;
```
Comment (pavelfeldman) [inline packages/web/src/components/toolbarButton.css:27]: We already do flex + align items, why is this needed?

File: packages/web/src/components/toolbarButton.css [hunk 6]
```diff
@@ -16,15 +16,25 @@
...
+  white-space: nowrap;
+  vertical-align: middle;
```
Comment (pavelfeldman) [inline packages/web/src/components/toolbarButton.css:29]: You should not use this style in flex.

File: packages/web/src/components/toolbarButton.tsx [hunk 7]
```diff
@@ -32,10 +33,11 @@ export const ToolbarButton: React.FC<React.PropsWithChildren<ToolbarButtonProps>
...
   disabled = false,
   toggled = false,
+  type = 'button',
   onClick = () => {},
 }) => {
   let className = `toolbar-button ${icon}`;
   if (toggled)
     className += ' toggled';
-  return <button className={className} onClick={onClick} title={title} disabled={!!disabled}><span className={`codicon codicon-${icon}`}></span>{ children }</button>;
+  return <button className={className} type={type} onClick={onClick} title={title} disabled={!!disabled}><span className={`codicon codicon-${icon}`}></span>{ children }</button>;
```
Comment (pavelfeldman) [inline packages/web/src/components/toolbarButton.tsx:42]: How can toolbar button be a submit / reset?

File: packages/web/src/components/toolbarButton.css (hunk 2 above)
Comment (debs-obrien) [reply packages/web/src/components/toolbarButton.css:24]: using the border radius of 6px as same as githubs buttons

File: packages/web/src/components/toolbarButton.css (hunk 3 above)
Comment (debs-obrien) [reply packages/web/src/components/toolbarButton.css:25]: these seems to be best color for border for both light and dark mode as inspired by github buttons

File: packages/web/src/components/toolbarButton.css (hunk 5 above)
Comment (debs-obrien) [reply packages/web/src/components/toolbarButton.css:27]: removed

File: packages/web/src/components/toolbarButton.css (hunk 6 above)
Comment (debs-obrien) [reply packages/web/src/components/toolbarButton.css:29]: removed

File: packages/web/src/components/toolbarButton.tsx (hunk 7 above)
Comment (debs-obrien) [reply packages/web/src/components/toolbarButton.tsx:42]: you're right

File: packages/web/src/components/toolbarButton.css (hunk 1 above)
Comment (debs-obrien) [reply packages/web/src/components/toolbarButton.css:37]: ok removed except for padding which is needed

**Review by pavelfeldman — APPROVED** [review]
